src/Cms/Auth.php

```diff
@@ -82,6 +82,7 @@ public function __construct(App $kirby)
     /**
      * Creates an authentication challenge
      * (one-time auth code)
+     * @since 3.5.1
```

Comment:
This method was already added in 3.5.0:

```suggestion
     * @since 3.5.0
```

Other methods in this class related to the same set of features:

- `enabledChallenges`: 3.5.1
- `login2fa`: 3.5.0
- `status`: 3.5.1
- `verifyChallenge`: 3.5.0

Reply:
Thanks, all added.